pkg/resources/digitalocean/cloud.go

```diff
@@ -76,6 +79,18 @@ func NewCloud(region string) (*Cloud, error) {
 	}, nil
 }
 
+func (c *Cloud) GetCloudGroups(cluster *kops.Cluster, instancegroups []*kops.InstanceGroup, warnUnmatched bool, nodeMap map[string]*v1.Node) (map[string]*fi.CloudGroup, error) {
+	return nil, fmt.Errorf("not implemented yet")
```

Comment:
From someone that has been bitten by this many times: it is nice to print fmt.Errorf("Cloud::GetCloudGroups not implemented yet") because otherwise you just see "not implemented yet" and you don't know which method it is!

Or you can panic() or glog.Fatalf, both of which give a stack trace

Reply:
Will update